### ISSUE TITLE
Interface fails to launch sometimes.

### ISSUE THREAD

**Issue by JeroenMathon** [JeroenMathon/Apherous#31]
Perhaps solving some of these errors can solve the problem.
```
(Apherous:26837): GLib-GObject-WARNING **: The property GtkWidget:margin-left is deprecated and shouldn't be used anymore. It will be removed in a future version.

(Apherous:26837): GLib-GObject-WARNING **: The property GtkWidget:margin-right is deprecated and shouldn't be used anymore. It will be removed in a future version.

(Apherous:26837): GLib-GObject-WARNING **: invalid uninstantiatable type '(null)' in cast to 'GstPipeline'

(Apherous:26837): GStreamer-CRITICAL **: gst_element_get_bus: assertion 'GST_IS_ELEMENT (element)' failed

(Apherous:26837): GStreamer-CRITICAL **: gst_bus_add_watch_full: assertion 'GST_IS_BUS (bus)' failed

(Apherous:26837): GStreamer-CRITICAL **: gst_mini_object_unref: assertion 'mini_object != NULL' failed

(Apherous:26837): GStreamer-CRITICAL **: gst_object_unref: assertion 'object != NULL' failed

(Apherous:26837): GLib-CRITICAL **: g_source_remove: assertion 'tag > 0' failed
```

**Comment by JeroenMathon** [comment]
Captured a core dump.
```
0x00007ffff466b500 in g_type_check_instance_cast () from /usr/lib/libgobject-2.0.so.0
```

**Comment by JeroenMathon** [comment]
The problem seems to be in the audioManager class.

**Comment by JeroenMathon** [comment]
The bug seems to appear at this call.
```
bus = gst_pipeline_get_bus(GST_PIPELINE(pipeline));
```
Which appears in [audio_mgr.cpp](https://github.com/JeroenMathon/Apherous/blob/gtk_interface/src/audio_mgr.cpp) on line 58

**Comment by JeroenMathon** [comment]
Fixed as of faab9d3dcda813741c6fd75be3e43dffbb0a2377.
pipeline was never initialized so  get_bus() got called on random memory.
By initializing the pipe line using.
```C++
gst_element_factory_make ("playbin", NULL)
```
I with the help of thaytan on the gstreamer IRC have fixed the problem.
